src/testdrive/src/action.rs

```diff
@@ -571,6 +578,7 @@ pub async fn create_state(
         kafka_config.set("bootstrap.servers", &config.kafka_addr);
         kafka_config.set("group.id", "materialize-testdrive");
         kafka_config.set("auto.offset.reset", "earliest");
+        kafka_config.set("isolation.level", "read_committed");
```

Comment:
is this an intentional change? im not familiar offhand with what this option does

Reply:
This means that a kafka consumer cannot see partial or aborted transactions when reading from a topic. Previously it didn't matter because we never wrote to a kafka topic transactionally, but now that we're using testdrive to validate eos it matters.

We may end up making it configurable if we want to write tests that check for aborts or exercise commit failure scenarios, but for now we can just set it to read_committed and be done with it.

Many clients default to enabled read_committed. I can't remember if librdkafka made that change or not, but easier to be explicit about it